test/primitives-test.coffee

```diff
@@ -146,6 +146,129 @@ describe 'Minim Primitives', ->
           foo: 'bar'
           z: 2
 
+  describe 'ElementType', ->
+    elType = undefined
+
+    before ->
+      elType = new minim.ElementType 'element'
+
+    describe '#getProperty', ->
+      it 'returns an error', ->
+        expect(elType.getProperty('foo').elementType()).to.equal 'error'
+
+    describe '#has', ->
+      it 'returns an error', ->
+        expect(elType.has('foo')).to.be.false
+
+    describe '#map', ->
+      it 'returns an error', ->
+        expect(elType.map((el) -> 'test').elementType()).to.equal 'error'
+
+    describe '#filter', ->
+      it 'returns an error', ->
+        expect(elType.filter((el) -> true).elementType()).to.equal 'error'
+
+    describe '#forEach', ->
+      it 'returns an error', ->
+        expect(elType.forEach((el) -> console.log('test')).elementType()).to.equal 'error'
```

Comment:
Not really. The console.log should never be called because `forEach` simply returns itself. I will change it because it may confuse.